### PR TITLE
Init document architecture.

### PR DESCRIPTION
**Pull request by dongzl**

### CODE REVIEW

**Review by chuntaojun — APPROVED** [review]

**Review by baerwang — APPROVED** [review]

**Review by Penglq — APPROVED** [review]

File: docs/01.intro.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+sidebar_position: 1
+---
+
+# Arana
+
+<div align=center>
+    <img src="https://raw.githubusercontent.com/arana-db/arana/master/docs/pics/arana-main.png"/>
+</div>
+
+`Arana` is a Cloud Native Database Proxy. It can be deployed as a Database mesh sidecar. It provides transparent data access capabilities,
```
Comment (AlexStocks) [inline docs/01.intro.md]: Database 这不是专有名词，小写即可。

File: docs/01.intro.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+|:-----------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|      [![Stargazers over time](https://starchart.cc/arana-db/arana.svg)](https://starchart.cc/arana-db/arana)      | [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=arana-db/arana)](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=arana-db/arana) |
+
+
+## Introduction | [中文](https://github.com/arana-db/arana/blob/master/README_CN.md)
+
+First, `Arana` is a Cloud Native Database Proxy. It provides transparent data access capabilities, when using `arana`,
+user doesn't need to care about the `sharding` details of database, they can use it just like a single `MySQL` database.
+`Arana` also provide abilities of `Multi Tenant`, `Distributed transaction`, `Shadow database`, `SQL Audit`, `Data encrypt / decrypt`
```
Comment (AlexStocks) [inline docs/01.intro.md]: `Arana` also provides many db abilities, such as

File: docs/01.intro.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+and so on. Through simple config, user can use these abilities provided by `arana` directly.
+
+Second, `Arana` can also be deployed as a Database mesh sidecar. As a Database mesh sidecar, arana switches data access from
+client mode to proxy mode, which greatly optimizes the startup speed of applications. It provides the ability to manage database
+traffic, it takes up very little container resources, doesn't affect the performance of application services in the container， but
+provides all the capabilities of proxy.
```
Comment (AlexStocks) [inline docs/01.intro.md]: it takes up very little container resources, and provides all the capabilities of proxy, while doesn't affect the performance of application services in the container.

File: docs/01.intro.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+## Contact
+
+Arana Chinese Community Meeting Time: **Every Saturday At 9:00PM GMT+8**
+
+<img src="https://raw.githubusercontent.com/arana-db/arana/master/docs/pics/dingtalk-group.jpeg" width="300px"/>
+
+## Contributing
+
+Thanks for your help improving the project! We are so happy to have you! We have a contributing guide to help you get involved in the Arana project.
```
Comment (AlexStocks) [inline docs/01.intro.md]: Thank you  for helping us improving the project!

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**